### PR TITLE
adjust hpwork and binary manager priority for priority policy

### PR DESCRIPTION
**Pull request by YiHyunJin**
Due to the priority policy, the priority of binary manager is 201 and
the binary manager loader is 200. However, hpwork must be running during
the binary manager loader. So binary manager is reprioritized to 203 and
hpwork to 201.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [8e5816167b059caedb4c8ead3b2e2ea7063319ee] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8e5816167b059caedb4c8ead3b2e2ea7063319ee] - Code Rule Check (C++) OK.

**Comment by jeongchanKim** [comment]
Please revert defconfig changes except which is related with loadable.

**Comment by seinfra** [comment]
Target : [9681be143c601003b0c16ec7b04ba97f52e1825c] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9681be143c601003b0c16ec7b04ba97f52e1825c] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [dfc04bbe981f5734470999e785de03c3e661c6e2] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [dfc04bbe981f5734470999e785de03c3e661c6e2] - Code Rule Check (C++) OK.

File: os/kernel/binary_manager/binary_manager.h [hunk 1]
```diff
@@ -49,7 +49,7 @@
 /* Binary Manager Core Thread information */
 #define BINARY_MANAGER_NAME        "binary_manager"           /* Binary manager thread name */
 #define BINARY_MANAGER_STACKSIZE   2048                       /* Binary manager thread stack size */
-#define BINARY_MANAGER_PRIORITY    201                        /* Binary manager thread priority */
+#define BINARY_MANAGER_PRIORITY    203                        /* Binary manager thread priority */
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager.h]: Could you add this in the title as well?

**Comment by sunghan-chang** [comment]
Please check wqueue.h.

**Comment by seinfra** [comment]
Target : [27b73f22d2cfc5dde54e89d8bf390dc26cd1abee] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [27b73f22d2cfc5dde54e89d8bf390dc26cd1abee] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [e90be0e37c0160b42cc7ef22f9a13fc4c081e229] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [e90be0e37c0160b42cc7ef22f9a13fc4c081e229] - Code Rule Check OK.

File: os/kernel/binary_manager/binary_manager.h (hunk 1 above)
Comment (YiHyunJin) [reply os/kernel/binary_manager/binary_manager.h]: yes

**Pull request merged**